File: EvenOrOdd.cpp
#include <string>

std::string even_or_odd(int number) 
{
  std::string Discover;
  if(number%2 == 0)
  {
    Discover = "Even";
  }
  else
  {
    Discover = "Odd";
  }
  return Discover;
}

File: EvenOrOdd.cs
using System;

namespace Solution
{
  public class SolutionClass
  {
    public static string EvenOrOdd(int number)
    {
      string Discover;
      if(number%2 ==0)
      {
        Discover = "Even";
      }
      else
      {
        Discover = "Odd";
      }
      return Discover;
    }
  }
}